sympy/printing/codeprinter.py

```diff
@@ -246,9 +246,7 @@ def _rate_index_position(self, p):
                                   "subclass of CodePrinter.")
 
     def _get_statement(self, codestring):
-        """Formats a codestring with the proper line ending."""
-        raise NotImplementedError("This function must be implemented by "
-                                  "subclass of CodePrinter.")
+        return "%s" % codestring
```

Comment:
We don't want to change to default behavior of the baseclass right now (`CodePrinter`), instead, override this method in `PythonCodePrinter`.